packages/updatePrescriptionStatus/tests/testDatabaseClient.test.ts

```diff
@@ -27,6 +27,7 @@ describe("Unit test persistDataItems", () => {
         PatientNHSNumber: "PatientNHSNumber_1",
         PharmacyODSCode: "PharmacyODSCode_1",
         PrescriptionID: "PrescriptionID_1",
+        RepeatNo: 1,
```

Comment:
These additions aren't needed in this test. The values are optional in the DataItem, and they don't affect the outcome of the test.

Reply:
The optional values have been removed.